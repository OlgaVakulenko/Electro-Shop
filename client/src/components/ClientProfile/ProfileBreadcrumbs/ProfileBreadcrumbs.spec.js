import React from 'react';
import { shallow } from 'enzyme';

import ProfileBreadcrumbs from './ProfileBreadcrumbs';

describe('The test of ProfileBreadcrumbs component', () => {
  it('ProfileBreadcrumbs was rendered', () => {
    const rendered = shallow(
      <ProfileBreadcrumbs />,
    );

    expect(rendered).toMatchSnapshot();
  });
});