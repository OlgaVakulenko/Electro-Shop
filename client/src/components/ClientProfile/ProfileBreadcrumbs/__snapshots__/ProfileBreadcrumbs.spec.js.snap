// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The test of ProfileBreadcrumbs component ProfileBreadcrumbs was rendered 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(Paper))
    className="makeStyles-paper-2"
    elevation={0}
  >
    <WithStyles(ForwardRef(Breadcrumbs))
      aria-label="breadcrumb"
      separator="â€º"
    >
      <WithStyles(ForwardRef(Link))
        className="makeStyles-breadCrumbLink-3"
        href="/"
      >
        Homepage
      </WithStyles(ForwardRef(Link))>
      <WithStyles(ForwardRef(Typography))
        color="textPrimary"
      >
        Profile
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(Breadcrumbs))>
  </WithStyles(ForwardRef(Paper))>
</div>
`;
